### ISSUE TITLE
Error in running demo of ARCS

### ISSUE THREAD

**Issue by yamanmalkoc** [bcgsc/arcs#61]
After I installed ARCS, I executed the ./runARCSdemo.sh script in the "Examples" subdirectory. The following error occurred while it was running.
```
Running ARCS...
Converting graph for LINKS...
Traceback (most recent call last):
  File "./makeTSVfile.py", line 96, in <module>
    readGraphFile(infile)
  File "./makeTSVfile.py", line 15, in readGraphFile
    with open(infile, 'r') as f:
IOError: [Errno 2] No such file or directory: 'hsapiens-8reformat.fa.scaff_s98_c5_l0_d0_e30000_r0.05_original.gv'
Running LINKS...
run complete
```

**Comment by lcoombe** [comment]
Hi Yaman -- Did these lines run properly?
```
echo Downloading sample Chromium read alignment .bam file and human genome assembly draft...
wget http://www.bcgsc.ca/downloads/supplementary/ARCS/testdata/NA24143_genome_phased_namesorted.bam1.sorted.bam
wget http://www.bcgsc.ca/downloads/supplementary/ARCS/testdata/hsapiens-8reformat.fa
```

I'll take a look with you tomorrow.

**Comment by lcoombe** [comment]
Issue fixed in 79ce11a
